api/cloudcontroller/ccv2/space_summary_test.go

```diff
@@ -0,0 +1,127 @@
+package ccv2_test
+
+import (
+	"net/http"
+
+	"code.cloudfoundry.org/cli/api/cloudcontroller/ccerror"
+	. "code.cloudfoundry.org/cli/api/cloudcontroller/ccv2"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/ghttp"
+)
+
+var _ = Describe("SpaceSummary", func() {
+	var client *Client
+
+	BeforeEach(func() {
+		client = NewTestClient()
+	})
+
+	Describe("GetSpaceSummary", func() {
+		When("there are no errors", func() {
+			BeforeEach(func() {
+				response := `{
+				 "name": "space-name",
+				 "apps": [
+						{
+							 "service_names": [
+									"service-instance-name"
+							 ],
+							 "name": "app-name"
+						}
+				 ],
+				 "services": [
+						{
+							 "name": "service-instance-name",
+							 "last_operation": {
+									"type": "create",
+									"state": "succeeded",
+									"description": "a description",
+									"updated_at": "some time",
+									"created_at": "some time"
+							 },
+							 "shared_from": null,
```

Comment:
It looks like these two fields (`shared_from`, `shared_to`) are declared and not asserted on in these tests. Do we need these fields then?

Reply:
Not needed. Removed.